Content.Client/Station/StationSystem.cs

```diff
@@ -5,7 +5,7 @@ namespace Content.Client.Station;
 /// <summary>
 /// This handles letting the client know stations are a thing. Only really used by an admin menu.
 /// </summary>
-public sealed class StationSystem : EntitySystem
+public sealed class StationSystem : SharedStationSystem
```

Comment:
```suggestion
public sealed partial class StationSystem : SharedStationSystem
```
let's do this while we are changing it anyways
makes things easier for forks